app.css

```diff
@@ -254,6 +254,7 @@ styling the header component
 
```

Comment:
### Your Popup
![popup-center-problem (copy)](https://user-images.githubusercontent.com/93039359/188851923-d014e327-9bea-45d7-b6cc-401c9ee9f5c8.png)

### Figma Popup
![popup-figma (copy)](https://user-images.githubusercontent.com/93039359/188851961-8b6824ca-6fce-4888-946a-92a45aa6336b.png)

- [ ] Your popup is not in the center as shown in the image above. Kindly, make it in the center to follow the Figma template :+1:
- [ ] Your popup window is missing the grey overlay as shown in the second image above. Please add it to make your design follow the Figma template :+1: